### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

**Comment by mulkieran** [comment]
Just an FYI...

**Review by drckeefe — APPROVED** [review]

**Review by bgurney-rh — APPROVED** [review]

**Comment by mulkieran** [comment]
All further changelog updates will be done as part of the release process...

File: CHANGES.txt [hunk 1]
```diff
@@ -1,3 +1,18 @@
+stratis-cli 2.4.1 (UNRELEASED)
+==============================
+
+- In blockdev listing, expose physical path if different from metadata path:
+  https://github.com/stratis-storage/stratis-cli/issues/754
+  https://github.com/stratis-storage/stratis-cli/pull/755
+
+- Add a small script to output GetManageObjects() result in JSON format:
```
Comment (bgurney-rh) [inline CHANGES.txt:8]: Should this say `GetManagedObjects()`?

Comment (mulkieran) [reply CHANGES.txt:8]: Whoops!

**Review by bgurney-rh — APPROVED** [review]

File: CHANGES.txt (hunk 1 above)
Comment (bgurney-rh) [reply CHANGES.txt]: OK; looks good.

**Pull request merged**